app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -98,10 +103,64 @@ class SuggestedEditsTasksFragment : Fragment() {
         setUpTasks()
         tasksRecyclerView.layoutManager = LinearLayoutManager(context)
         tasksRecyclerView.adapter = RecyclerAdapter(displayedTasks)
+        setUpTooltips()
 
         clearContents()
     }
 
+    private fun setUpTooltips() {
+        val balloon = createBalloon(requireContext()) {
+            setArrowSize(10)
+            setWidthRatio(0.5f)
+            setHeight(65)
+            setArrowPosition(0.7f)
+            setCornerRadius(4f)
+            setAlpha(0.9f)
+            setText("Edit quality.")
+            setTextColorResource(R.color.base100)
+            setTextIsHtml(true)
+            setIconDrawable(ContextCompat.getDrawable(requireContext(), R.drawable.ic_baseline_person_24))
+            setBackgroundColorResource(R.color.accent30)
+            setBalloonAnimation(BalloonAnimation.FADE)
+            setLifecycleOwner(lifecycleOwner)
+        }
+        val balloon1 = createBalloon(requireContext()) {
+            setArrowSize(10)
+            setWidthRatio(0.5f)
+            setHeight(65)
+            setArrowPosition(0.7f)
+            setCornerRadius(4f)
+            setAlpha(0.9f)
+            setText("EditStreakStatsView")
+            setTextColorResource(R.color.chip_text_color_light)
+            setTextIsHtml(true)
+            setIconDrawable(ContextCompat.getDrawable(requireContext(), R.drawable.ic_baseline_person_24))
+            setBackgroundColorResource(R.color.accent30)
+            setBalloonAnimation(BalloonAnimation.FADE)
+            setLifecycleOwner(lifecycleOwner)
+        }
+        val balloon2 = createBalloon(requireContext()) {
+            setArrowSize(10)
+            setWidthRatio(0.5f)
+            setHeight(65)
+            setArrowPosition(0.7f)
+            setCornerRadius(4f)
+            setAlpha(0.9f)
+            setText("ContributionsStatsView")
+            setTextColorResource(R.color.base100)
+            setTextIsHtml(true)
+            setIconDrawable(ContextCompat.getDrawable(requireContext(), R.drawable.ic_baseline_person_24))
+            setBackgroundColorResource(R.color.accent30)
+            setBalloonAnimation(BalloonAnimation.FADE)
+            setLifecycleOwner(lifecycleOwner)
+        }
+        editQualityStatsView.showAlignTop(balloon)
+        balloon
```

Comment:
Looking for a better way to do this sequencing... just for readability...  also just a note that it is possible to define the dismiss listener.. I have just not used it because the sequencing logic overrides it..